templates/contribute/naming.md

```diff
@@ -416,3 +416,24 @@ automatically generated unidirectional implications, named `.inj`,
 and there is no intention to change this.
 When such an automatically generated lemma already exists,
 and a bidirectional lemma is needed, it may be named `.inj_iff`.
+
+### Inductive and recursive principles
+
+Inductive/recursive principles are ways to construct data or proofs for all elements of some type `T`,
+by providing ways to construct this data or proof in more constrained specific contexts. 
+These principles generally are phrased to first accept a *motive*,
```

Comment:
```suggestion
These principles generally are phrased to first accept a `motive` argument,
```
since the argument is not only a motive, but should be _called_ `motive`.

Reply:
that isn't consistently done over all of mathlib though. for example, check out [Quotient.inductionOn₂](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Quot.html#Quotient.inductionOn%E2%82%82%27) for an example in Mathlib, or [WellFounded.induction](https://leanprover-community.github.io/mathlib4_docs/Init/WF.html#WellFounded.induction) in Init

Reply:
additionally, i think that at least for `Prop`-typed motives, naming the variable `P` for predicate is clearer, particularly since the term "motive" might be unfamiliar to new members. (i myself found it intimidating, as i hadn't come across the notion in my maths undergrad.) because of this, i'd rather emphasise that it is a technical term than that it is the name commonly used for such a variable.

Reply:
my opinion right now also extends to `Type`-typed motives, but at least `Prop` gives a clear alternative.

Reply:
You're right, `P` probably is clearer for `Prop` than `motive`. Can you post that suggestion in the zulip thread to get consensus?